templates/registration/logged_out.html

```diff
@@ -0,0 +1,4 @@
+{% extends "base.html" %}
+
+{% block content %}
+{% endblock %}
```

Comment:
The `content` block is empty. Consider adding a message to inform users that they have successfully logged out or provide a link to log back in or return to the home page.